EX2/app/src/main/java/com/example/ex2/AliveActivity.java

```diff
@@ -0,0 +1,33 @@
+package com.example.ex2;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.os.Bundle;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AliveActivity extends AppCompatActivity {
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_alive);
+
+        List<Personality> alive = new ArrayList<>();
+        Personality Adi = new Personality(R.drawable.mere1, "Meredith");
```

Comment:
Redundant and should be removed (I guess it's a leftover?)

Reply:
totally, deleted